extensions/DPLforum/DPLforum_body.php

```diff
@@ -371,31 +371,54 @@ function parse( &$input, Parser $parser ) {
 			}
 		} elseif ( is_null( $title ) ) {
 			while ( $row = $dbr->fetchObject( $res ) ) {
-				if( isset( $row->first_time ) ) {
+				if ( isset( $row->first_time ) ) {
 					$first_time = $row->first_time;
 				} else {
 					$first_time = '';
 				}
 
-				if( isset( $row->first_user ) ) {
-					$first_user = $row->first_user;
-				} else {
-					$first_user = '';
+				$first_user = '';
+				if ( isset( $row->first_user ) || isset( $row->first_user_text ) ) {
+					$firstUser = is_int( $row->first_user ) ? $row->first_user : 0;
```

Comment:
I would not rely on `is_int()` here.  Sometimes data types in PHP might not be properly mapped with SQL types and then everything becomes `string`.  I would use `is_numeric()` which will work consistently for both integers and strings, e.g.:

```
michal@dev-michal:~$ php -n -r "var_dump(is_int('312'));"
bool(false)
michal@dev-michal:~$ php -n -r "var_dump(is_numeric('312'));"
bool(true)
michal@dev-michal:~$ php -n -r "var_dump(is_numeric(312));"
bool(true)
```